.gitignore

```diff
@@ -1,6 +1,7 @@
-.idea/*
+.idea
 composer.lock
-vendor/
+vendor
 builds/
 tests/vendor/
-go.sum
\ No newline at end of file
+.rr.yaml
```

Comment:
why is this file added, its default template which is used to init config in some apps, leaving it out of the repository won't break anything but will have new users ux

Reply:
Default config already committed into the repo. Since .rr.yaml is just a sample of config there are 2 options:
1. Put this file into .gitignore and allow users to make their own local changes in file
2. Remove it from .gitignore, but in this way, I have to create a copy of that file for local testing for example.
So, which variant would you prefer?

Reply:
1.